rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -184,16 +184,6 @@
             "form": {
               "templateOptions": {
                 "doNotSubmitOnEnter": true
-              },
-              "validation": {
```

Comment:
Seems a bad idea. The frontend should be able to do the validation. You can probably do this in a similar way than the locations: https://github.com/rero/rero-ils/blob/469b76f275b535c61dd634cffefe8213eb13d931/rero_ils/modules/locations/jsonschemas/locations/location-v0.0.1.json#L88.

Reply:
There is no organisation pid in the data model of the patron so I'm not sure how to filter.